app/jobs/store_version_contents_job.rb

```diff
@@ -0,0 +1,39 @@
+class StoreVersionContentsJob < ApplicationJob
+  include GoodJob::ActiveJobExtensions::Concurrency
+  good_job_control_concurrency_with total_limit: 1, key: -> { "store-contents-#{version_arg.full_name}" }
+  queue_as :default
+  self.queue_adapter = :good_job
```

Comment:
This is not needed since https://github.com/rubygems/rubygems.org/commit/c791b89bb38a996e773c9a91dc0f93ea9e0560be.

Reply:
Gotcha. Funny enough an earlier review had me add it. We've been editing this pr for a while.

Reply:
Fixed